src/State/SubsiteState.php

```diff
@@ -48,6 +49,9 @@ public function getSubsiteId()
      */
     public function setSubsiteId($id)
     {
+        if (!is_int($id) && !is_null($id)) {
+            Deprecation::notice('2.8.0', 'Passing multiple IDs is deprecated, only pass a single ID instead.');
+        }
```

Comment:
A string `"1"` would still be valid and not deprecated - or if it would be deprecated, it shouldn't have this message.

Reply:
Updated